prody/tests/database/test_pfam.py

```diff
@@ -26,63 +45,30 @@ def setUpClass(cls):
                        'hellow', 'hello']
 
     def testUniprotAccMulti(self):
-        """Test the outcome of a simple search scenario using a Uniprot Accession
```

Comment:
No need to delete doc strings like this

Reply:
I had it add back in doc strings and comments from original.